frontend/src/components/searchBar/MessageSearchBar.tsx

```diff
@@ -3,20 +3,24 @@ import { SearchContainer, SearchTypeSwitcher } from "./MessageSearchBar.styled";
 import Search from "components/search/Search";
 import useMessageSearchBar from "./useMessageSearchBar";
 
-interface Props {
-    searchMode: string;
-    handleSetSearchMode: () => void;
-}
-
-const MessageSearchBar: FC<Props> = ({ searchMode, handleSetSearchMode }) => {
-    const { handleSetSearchParams } = useMessageSearchBar();
+const MessageSearchBar: FC = () => {
```

Comment:
unnecessary Type 'FC' this @Skolaczk write in other comments section couple time ago, check all components